### ISSUE TITLE
Nano S+ FW Version 1.3.0 Support

### ISSUE THREAD

**Issue by harrychin** [LedgerHQ/ledger-device-rust-sdk#224]
It looks like the latest firmware update for the Nano S+, 1.3.0, has vaulted the API Level for the Nano S+ from 5 to 22, could we please get an update on when we can expect the SDK to get pulled up to support this change?

It doesn't seem quite as straightforward as just pulling up the values [here](https://github.com/LedgerHQ/ledger-device-rust-sdk/blob/master/ledger_secure_sdk_sys/build.rs#L223).

**Comment by yogh333** [comment]
@harrychin Yes indeed Rust SDK needs to be updated but in the meantime updating the value as shown above should be enough. I assume that you work out of Docker container but keep also in mind that we provide [Docker images](https://github.com/LedgerHQ/ledger-app-builder) that include everything needed to develop device apps (in C or in Rust) and for instance up to date C SDK.

**Comment by yogh333** [comment]
Fixed by #226
